### ISSUE TITLE
Invalid certificate when not using OpenshiftSDN for networking

### ISSUE THREAD

**Issue by tmjd** [openshift/installer#2031]
# Version

```console
$ openshift-install version
./openshift-install-4.1.4 v4.1.4-201906271212-dirty
built from commit bf47826c077d16798c556b1bd143a5bbfac14271
release image quay.io/openshift-release-dev/ocp-release@sha256:a6c177eb007d20bb00bfd8f829e99bd40137167480112bd5ae1c25e40a4a163a
```

# Platform:
AWS

# What happened?
When using Calico for networking many pods that access the kubernetes apiserver report 
`x509: cannot validate certificate for 172.30.0.1 because it doesn't contain any IP SANs`.
Operators that are using the external api DNS name are able to connect to the apiserver successfully but not any that are using the kubernetes service.
While the API server is being hosted on the bootstrap node I believe this error does not happen and it is only after the apiserver starts running on the master nodes.

# What you expected to happen?

I expect operators and pods to still be able to access the API server.

# How to reproduce it (as minimally and precisely as possible)?

I don't have a simple reproduction currently.

# Anything else we need to know?

This seems to not be consistent. A few times my cluster has worked as expected.

I am happy to create a cluster and do some debugging but I do not know what to dig or poke to figure this out.

# References

**Comment by lmm** [comment]
I've also ran into this with certificate validation issue with previously working versions of openshift-installer like v4.1.2

**Comment by tmjd** [comment]
What operator or component should be updating the certs to include the kubernetes service IP? I could do a little more digging if someone could direct me where to look.

**Comment by tmjd** [comment]
@squeed would you have any suggestions where to look?

**Comment by squeed** [comment]
Third-party network operators need to make sure they publish the current enabled CIDRs the "Network.config.openshift.io Status` field - see [here](https://github.com/openshift/api/blob/release-4.1/config/v1/types_network.go#L21).

That will cause the APIServer's certificate to be updated.

**Comment by tmjd** [comment]
Thanks.
Due to a misconfiguration on our/my part our operator was not getting to the point where it would update the Network Status.

One note, I did fix an installation that had gotten into the state where the IP Sans were not present and I fixed the operator so it updated the Status but the IP Sans problem did not seem to resolve. I ended up just rebuilding my cluster and with the fixed config it worked then.
